src/components/popups/JobOffers.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import { useDispatch } from "@/hooks/useTypedDispatch";
+import { toggleBodyScrolling } from "@/store/feature/ui.slice";
+import Popup from "../ui/Popup";
+import Button from "../ui/button";
+import CloseIcon from "@/icons/close-icon.svg";
+import JobIcon from "@/icons/briefcase.svg";
+
+/**
+ * JobOffers interface props
+ *
+ * @interface JobOffersProps
+ * @typedef {JobOffersProps}
+ */
+interface JobOffersProps {
+  externalClick?: () => void;
+}
+
+/**
+ * JobOffers component
+ *
+ * @export
+ * @param {JobOffersProps} param0
+ * @param {() => void} param0.externalClick
+ * @returns {*}
+ */
+export default function JobOffers({ externalClick }: JobOffersProps) {
+  const [show, setShow] = useState(false);
+
+  const dispatch = useDispatch();
+
+  const onOpenJobOffers = () => {
```

Comment:
```suggestion
  const toggle = () => {
```

Since its what we are using on the popups as a standard.